packages/components/src/button/button.tsx

```diff
@@ -135,6 +140,28 @@ const styles = cva({
       true: 'aspect-square !px-0',
     },
   },
+  compoundVariants: [
+    {
+      size: ['40', '32'],
+      placement: 'before',
+      className: 'pl-3 pr-4',
+    },
+    {
+      size: ['40', '32'],
+      placement: 'after',
+      className: 'pl-4 pr-3',
+    },
+    {
+      size: '24',
+      placement: 'before',
+      className: 'pl-2 pr-3',
+    },
+    {
+      size: '24',
+      placement: 'after',
+      className: 'pl-3 pr-2',
+    },
```

Comment:
'40' size is correct, but '32' should have 8px and 12px and 23 has 6px and 8px on paddings.